src/components/Holdings/HoldingModal.tsx

```diff
@@ -116,6 +127,15 @@ const HoldingModal = (props: ModalProps) => {
     props.onCreate(holding)
   }
 
+  const deleteHolding = () => {
+    if (props.holding) {
+      db.get('holdings')
+      .remove({id : props.holding.id})
+      .write()
+    } 
+    setRedirect(true)
```

Comment:
Can we redirect directly from here without rendering the redirect component?

Reply:
Yes, in order to do this, I had to pull in the `withRouter` component so that `HoldingModal` could have the props necessary to access history to push `/` onto the stack.